### PR TITLE
Bump github.com/containerd/containerd from 1.6.18 to 1.6.26

### PR DESCRIPTION
**Pull request by denyeart**
Bump github.com/containerd/containerd from 1.6.18 to 1.6.26
Only move golang.org/x/tools to v0.14.0 (later versions introduce an issue with ./scripts/metrics_doc.sh check)
Only move google.golang.org/protobuf to v1.31.0 (later versions introduce an issue with chaincode build)

Update tests to reflect message updates in latest grpc v1.61.0.

### CODE REVIEW

**Comment by pfi79** [comment]
https://github.com/hyperledger/fabric/pull/4668#issuecomment-1928002656

**Comment by pfi79** [comment]
https://github.com/hyperledger/fabric/pull/4668#issuecomment-1928010077

File: integration/smartbft/smartbft_test.go [hunk 1]
```diff
@@ -1643,7 +1643,8 @@ var _ = Describe("EndToEnd Smart BFT configuration test", func() {
 			// If backoff's default settings are left, it will take 20-25 seconds after the 7th attempt to reach the 8th attempt.
 			// If the backoff is made controllable, as here in the test, the maximum time between attempts will be 3 seconds.
 			for i := 0; i < 7; i++ {
-				Eventually(ordererRunners[0].Err(), network.EventuallyTimeout, time.Second).Should(gbytes.Say("pickfirstBalancer: UpdateSubConnState.*IDLE connection error"))
+				// look for grpc Info message
+				Eventually(ordererRunners[0].Err(), network.EventuallyTimeout, time.Second).Should(gbytes.Say("Received SubConn state update.*IDLE connection error"))
```
Comment (pfi79) [inline integration/smartbft/smartbft_test.go]: `IDLE ConnectionError`

Comment (denyeart) [reply integration/smartbft/smartbft_test.go]: Thank you. Relying on these log messages is fragile!

**Comment by pfi79** [comment]
integration/gateway/gateway_bft_test.go line 183 
need to be replaced
```
"%s, \\{READY"
```
on
```
"%s, \\{ConnectivityState:READY"
```

File: integration/smartbft/smartbft_test.go (hunk 1 above)
Comment (pfi79) [reply integration/smartbft/smartbft_test.go]: Totally agree with you. 
But I can't think of any other way to show and prove that there are problems with an unconfigured backoff grpc.

**Comment by denyeart** [comment]
> integration/gateway/gateway_bft_test.go line 183 need to be replaced
> 
> ```
> "%s, \\{READY"
> ```
> 
> on
> 
> ```
> "%s, \\{ConnectivityState:READY"
> ```

Ah yes, hopefully this is the last one.

File: integration/gateway/gateway_bft_test.go [hunk 2]
```diff
@@ -113,7 +113,7 @@ var _ = Describe("GatewayService with BFT ordering service", func() {
 		os.RemoveAll(testDir)
 	})
 
-	It("Submit transaction", func() {
+	FIt("Submit transaction", func() {
```
Comment (pfi79) [inline integration/gateway/gateway_bft_test.go]: Maybe we should put `It' back in.
or
GINKGO_EDITOR_INTEGRATION=true before running focus test

Comment (denyeart) [reply integration/gateway/gateway_bft_test.go]: Ready to go now, thanks for the help!

**Review by ale-linux — APPROVED** [review]

**Pull request merged**